bot/cogs/bot.py

```diff
@@ -60,6 +62,20 @@ def __init__(self, bot: AutoShardedBot):
 
         await ctx.invoke(self.info)
 
+    async def on_message(self, msg: Message):
+        """
+        Auto codeblock python code
+        """
+        if msg.content.count("\n") >= 3:  # more than three lines
+            try:
+                tree = ast.parse(msg.content)
+                # tries to parse the code into a valid ast if it throws a SyntaxError it is no valid python
+                if not all(isinstance(node, ast.Expr) for node in tree.body):  # we dont want hi\nthere\nguys\nD
+                    await self.bot.send_message(msg.author, "```python\n{}\n```".format(msg.content))
```

Comment:
send_message() is pre-rewrite, and I'm pretty sure you want to send it to the channel, so the rewrite version would be msg.channel.send(<message here>)
Also if the message (with the unformatted python code) were >1986 characters this line would throw an error, maybe add a check against this?
(reposting this comment because the old one shows up as outdated)